sites/goland-guide/contents/tips/implement-an-interface/index.md

```diff
@@ -0,0 +1,19 @@
+---
+type: tip
+date: 2020-02-05
+title: Implement an interface
+technologies: [go]
+topics: [editing]
+author: dlsniper
+subtitle: Implementing an interface is a common task that Go programmers need to do
+thumbnail: ./thumbnail.png
+cardThumbnail: ./card.png
+shortVideo:
+  poster: ./poster_short.png
+  url: https://www.youtube.com/watch?v=0ChuCzfdcSE&list=PLM-t1Z4tbFflkIOaap4P-BV30ZrZwrDld&index=2
+leadin: |
+    GoLand offers the option to quickly implement an interface using the _Implement 
+    methods_ action, _Ctrl + I_ on Windows/Linux or _Shift + I_ on macOS, and then select
```

Comment:
For the actions, I was doing a style more like Action Name (blah Win/Linux, blah macOS).

Reply:
This is something I would like to change. What I want to have is a way to define the shortcut and then have the user select their OS/keymap, like we have for the web help documentation.